sre/storage/kvStorage_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPutandGet(t *testing.T) {
+
+	m := Map{}
+	tables := []struct {
+		x string
+		y interface{}
+	}{
+		{"a", 1},
+		{"b", 2.0},
+		{"c", "harry"},
+	}
+	for _, table := range tables {
+		m.Put(table.x, table.y)
+		val, _ := m.Get(table.x)
+		// assert.EqualError(t, err, nil)
```

Comment:
uncomment

Reply:
assert.equalerror()  will  pass in test cases when things goes right that means it will throw error